container/shipit/kubernetes/engine.py

```diff
@@ -33,19 +40,41 @@ def save_config(self):
         dest_path = os.path.join(self.base_path, SHIPIT_PATH, SHIPIT_CONFIG_PATH, self.name)
         create_path(dest_path)
 
-        templates = Service(config=self.config, project_name=self.project_name).get_template()
-        for template in templates:
-            name = "%s-service.json" % template['metadata']['name']
-            with open(os.path.join(dest_path, name), 'w') as f:
-                f.write(json.dumps(template, indent=4))
+        with make_temp_dir() as temp_dir:
+            with open(os.path.join(temp_dir, 'docker-compose.yml'), 'w') as f:
+                f.write(yaml.safe_dump(self.config.get('services')))
 
-        templates = Deployment(config=self.config, project_name=self.project_name).get_template()
-        for template in templates:
-            name = "%s-deployment.json" % template['metadata']['name']
-            with open(os.path.join(dest_path, name), 'w') as f:
-                f.write(json.dumps(template, indent=4))
+            jinja_render_to_temp('shipit-kompose-docker-compose.j2.yml',
+                                 temp_dir,
+                                 'shipit-kompose-compose.yml',
+                                 builder_img_id="containscafeine/{}:{}".format(
```

Comment:
we're going to need to fix this to be not from your account. either that or we actually just install kompose into the ansible-container builder image - it will be in epel soon so we can just install from rpm. 

@chouseknecht, thoughts?